# CustomService.srv
---
string request_data
int32 response_value
